util/Pos/Util/Orphans.hs

```diff
@@ -192,46 +189,6 @@ instance
 
 deriving instance LiftLocal LoggerNameBox
 
-instance {-# OVERLAPPABLE #-}
```

Comment:
Ah, I see. `networking` now depends on `util` so this code had to move somewhere else. If that's the case, wouldn't `networking` be a better home for it? Or is there some other reason that prevents that?

Reply:
I've ditched the mockable stuff altogether for networking, in a commit that's in `networking/develop` but has not yet made it to review against `develop`.

Reply:
The mockable stuff (and hence all of these orphan instances) never really had to do with networking, it was an accident that it ended up there. I think it's OK to dump it into lib, as I plan to and will get rid of it. How soon that happens just depends upon how fast we can review and merge my work :)

Reply:
Make sure to tag me. I want to see that stuff disappear :).